conda-store-server/tests/conftest.py

```diff
@@ -173,6 +177,29 @@ def simple_conda_lock():
         return yaml.safe_load(f)
 
 
-@pytest.fixture
-def current_prefix():
-    return pathlib.Path(os.environ["CONDA_PREFIX"])
+@pytest.fixture(
```

Comment:
Love it. This is a great way I think to be able to test more complete environment in the future.